deployment/ansible/development/roles/pycharm/templates/run-manager-environment.xml.j2

```diff
@@ -4,24 +4,12 @@
         <env name="DISPLAY" value=":99" />
         <env name="INASAFE_LOCALE" value="id" />
         <env name="C_FORCE_ROOT" value="True" />
+        <env name="TASK_ALWAYS_EAGER" value="True" />
```

Comment:
@ismailsunni Make sure you have TASK_ALWAYS_EAGER environment variable. Set it to True here so you can run unittest synchronously.